.travis.yml

```diff
@@ -36,4 +36,5 @@ script:
  - ./.buildifier.sh
  - python -m unittest discover -s reconciletags -p 'reconciletags_test.py'
  - bazel clean && bazel build //...
+ - sudo rm -rf /etc/docker
```

Comment:
there's actually only really one file in there, but yes, Dan said to remove the whole folder